### ISSUE TITLE
Subpages of Cookbook are not listed in navigation sidebar

### ISSUE THREAD

**Issue by crtl** [flutter/website#7245]
### Describe the problem

View any article in the "Cookbook" section like https://docs.flutter.dev/cookbook/testing/widget/introduction

In the sidebar, only "Cookbook" link is marked as active and it does not have a dropdown / nested menu so when clicking on it users only reach the cookbook root side.
To get an overview of topics in the cookbook it is required to click the breadcrumb links in the page or manually paginate throgh the pages.

### Expected fix

The breadcrumb structure of "Cookbook" should be present in the sidebar too to allow easy and fast navigation on the site and get a quick overview of topics to come without leaving the current page.
This should also apply to all other navigation links in the "Samples and Tutorials" section of the navigation.

### Additional context

Navigation in the "Development" section of the sidebar already has the desired end result where you can visit each link from the sidebar directly.

Basically all links from the index page https://docs.flutter.dev/cookbook should be present in the sidebar.

**Comment by Hirdey-1999** [comment]
I Can Work With This Issue But Can Navigate Me To The Folder For The CodeBase Of This Particular Issue
